src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs

```diff
@@ -1,13 +1,16 @@
-﻿using System;
+﻿// FindFirstDescendent is not available in UWP
```

Comment:
in toolkit, use **Get**FirstDescendent which is an any-plat impl
the Find... from uno.ui is uno-only

Reply:
either case, this seems wrong way to fix the problem?

Reply:
Small note here, `FindFirstDescendent` from Uno.UI is both Uno + WinAppSDK (but not UWP)

https://github.com/unoplatform/uno/blob/9450602cda0194514512ba9572dfc7f745fdfae0/src/Uno.UI.Toolkit/Uno.UI.Toolkit.Windows.csproj#L31-L36